content/millennium/r4/base/management/encounter.md

```diff
@@ -225,6 +227,12 @@ Create an individual Encounter.
 
     POST /Encounter
 
+_Implementation Notes_
+
+* Only one [Encounter.location.location] reference is permitted.
+* [Encounter.serviceProvider] and [Encounter.location.location] are mutually exclusive. Only one or the other is permitted.
```

Comment:
How about rewording this to something like "[Encounter.location.location] cannot be provided when [Encounter.serviceProvider] is valued." We could also have a similar note in the implementation notes for serviceProvider field.

Reply:
changed wording https://github.com/cerner/fhir.cerner.com/pull/803/commits/1d0f2634bd131127317a63f5458e2f27fc579829#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR553 and also added to serviceProvider https://github.com/cerner/fhir.cerner.com/pull/803/commits/1d0f2634bd131127317a63f5458e2f27fc579829#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR532